tests/Report/junit.xsd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
And we need it right? So next reason to store it outside tests and point to it instead of external resource.